packages/bsky/src/auth.ts

```diff
@@ -76,7 +76,19 @@ export const getJwtStrFromReq = (req: express.Request): string | null => {
 }
 
 const parseB64UrlToJson = (b64: string) => {
-  return JSON.parse(ui8.toString(ui8.fromString(b64, 'utf8')))
+  return JSON.parse(common.b64UrlToUtf8(b64))
+}
+
+const parsePayload = (b64: string): JwtPayload => {
+  const payload = parseB64UrlToJson(b64)
+  if (typeof payload !== 'object') {
```

Comment:
If for some reason the JSON were `null` 🙃 
```suggestion
  if (!payload || typeof payload !== 'object') {
```

Reply:
ugh dammit you right 🙇